src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -656,15 +656,15 @@ private bool CheckFieldValueKind(SyntaxNode node, BoundFieldAccess fieldAccess,
         private static bool CheckSimpleAssignmentValueKind(SyntaxNode node, BoundAssignmentOperator assignment, BindValueKind valueKind, DiagnosticBag diagnostics)
         {
             // Only ref-assigns produce LValues
-            if (assignment.RefKind != RefKind.None)
+            if (assignment.IsRef)
```

Comment:
>IsRef [](start = 27, length = 5)

Do we now have both properties IsRef and RefKind on BoundAssignmentOperator?

Reply:
No, I've removed RefKind in favor of IsRef, so this brings the bound node into agreement with your IOperation design.